apps/bsd/src/screens/DashboardScreen.tsx

```diff
@@ -116,6 +154,32 @@ const DashboardScreen: React.FC<Props> = ({
           <p>No ballots have been scanned.</p>
         )}
       </Prose>
+      {pendingDeleteBatchId && (
+        <Modal
+          centerContent
```

Comment:
nit but I usually add `onOverlayClick={cancelDeleteBatch}` to the Modal props which will allow the user to click outside the modal to dismiss it in addition to pressing the close button. It might complicate things here a bit though as you would only want to have that when its not in the "Deleting..." state.

Reply:
Good suggestion, added.